src/vstart.sh

```diff
@@ -661,7 +661,9 @@ if [ "$start_osd" -eq 1 ]; then
 EOF
 
 	    rm -rf $CEPH_DEV_DIR/osd$osd || true
-	    for f in $CEPH_DEV_DIR/osd$osd/*; do btrfs sub delete $f &> /dev/null || true; done
+            if [ -x /usr/bin/btrfs ]; then
```

Comment:
`btrfs` is considered as a system administration command, and usually lives in `/usr`. probably you can use `command -v` instead to avoid this fuss? i.e.
```sh
if command -v btrfs > /dev/null; then
  ...
fi
```